### ISSUE TITLE
Freecam Cheat

### ISSUE THREAD

**Issue by Konclan** [ChaosInitiative/Portal-2-Community-Edition#81]
A way to detach your camera from your player body, when not in thirdperson it will show your viewmodel but in thirdperson it will show the player model. Scrollwheel will change the fly speed so the player can easily maneuver the camera. This has several applications and would be more accessible for people who don't know/ don't want to use demos. (perhaps a way to continue controlling the character while still in freecam would be nice as well)

**Comment by L89David** [comment]
ins't this the same thing as #78?

**Comment by Konclan** [comment]
No, screenshot mode wouldn't be a cheat and only toggles the hud. This would be a cheat as it allows you to go anywhere in the map. However it might also be nice to have a freecam that isn't a cheat but this camera can't go through the world. Maybe like a deployable robot or something. This would allow for people in tests to see stuff they can't see from certain areas. Not actually sure if that is worth adding though.

**Comment by L89David** [comment]
yeah just noticed it, that's why I removed the comment... In my opinion, I don't think that giving the players a way to inspect the level with "intended ways" would be really good...

**Comment by Konclan** [comment]
I mean it would be a choice the map maker would have, like a tool you can give the player.

**Comment by L89David** [comment]
oh ok, that changes the things

**Comment by N5F** [comment]
> 



> yeah just noticed it, that's why I removed the comment =P... In my opinion, I don't think that giving the players a way to inspect the level with "unintended ways" would be really good...

they already can on og p2 with noclip so i dont think there should be a problem with that
